.drone.star

```diff
@@ -546,7 +546,17 @@ def main(ctx):
     after = afterPipelines(ctx)
     dependsOn(stages, after)
 
-    return before + stages + after + checkStarlark()
+    if ctx.build.event == "cron":
+        return example_deploys(ctx)
```

Comment:
@wkloucek this `if` statement now means that the nightly cron job does not run the full CI. It is just running `example_deploys`.

Was that intended?

Reply:
Was not intended :-/  -> revert in https://github.com/owncloud/web/pull/5199